main.py

```diff
@@ -52,19 +56,20 @@ def show_all_books():
 
 
 def search_book():
-    search_term = ui.ask_question('Enter search term, will match partial authors or titles.')
+    search_term = ui.ask_question(
+        'Enter search term, will match partial authors or titles.')
     matches = store.book_search(search_term)
     ui.show_books(matches)
 
 
 def change_read():
 
     book_id = ui.get_book_id()
-    book = store.get_book_by_id(book_id)  
-    new_read = ui.get_read_value()     
-    book.read = new_read 
+    book = store.get_book_by_id(book_id)
```

Comment:
whitespace changes add clutter - try to avoid.  if there's issues with whitespace deal with it under another issue, branch, commit(s)

Reply:
I will update this one under a different branch.